state-chain/pallets/cf-swapping/src/lib.rs

```diff
@@ -359,6 +359,13 @@ impl<T: Config> From<DispatchError> for BatchExecutionError<T> {
 	}
 }
 
+#[derive(Clone, Copy, Debug, Encode, Decode, TypeInfo, PartialEq, Eq)]
+pub enum SwapRequestCompletionReason {
+	Aborted,
+	Expired,
+	Executed,
```

Comment:
As discussed in the Linear issue, there are some subtle ambiguities in this, would be good to add some clarifying doc comments.

(eg. When we stop executing because of price limits, is it Aborted or Executed? etc..)